tools/roxvet/analyzers/validateimports/analyzer.go

```diff
@@ -19,27 +20,27 @@ const doc = `check that imports are valid`
 const roxPrefix = "github.com/stackrox/rox/"
 
 var (
-	validRoots = []string{
```

Comment:
Would it make sense to add a comment here requesting the set items to be kept in alphabetic order ?